cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -334,13 +334,30 @@ def add_custom_error_metrics(self):
         error_metric_dict = {
             "Other Error and Warning Messages": other_potential_issues,
             "Jobs Not Starting Errors": jobs_not_starting_errors,
-            "Unexpected EC2 Termination": compute_node_events,
+            "Issues with EC2 Instances": compute_node_events,
```

Comment:
It's a bit hard to keep track of the keys being used here.
You could use a nampedtuple e.g.
```
ErrorMetric = namedtuple("ErrorMetric", "title metric_filters troubleshooting_link")
...
error_metrics = [
	ErrorMetric(
		title="Issues with EC2 Instances",
		metric_filters=compute_node_events,
		troubleshooting_link="Issues with EC2 [Troubleshooting Resources](https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting.html)"
	),
	...
]
...

for title, metric_filters, troubleshooting_link in error_metrics:
	...

```

That way you can reference the attributes instead of an index.